doc/rule-descriptions.md

```diff
@@ -46,6 +46,7 @@
 | meta-viewport | Ensures &lt;meta name=&quot;viewport&quot;&gt; does not disable text scaling and zooming | cat.sensory-and-visual-cues, wcag2aa, wcag144 | true |
 | object-alt | Ensures &lt;object&gt; elements have alternate text | cat.text-alternatives, wcag2a, wcag111, section508, section508.22.a | true |
 | p-as-heading | Ensure p elements are not used to style headings | cat.semantics, wcag2a, wcag131, experimental | true |
+| page-has-heading-one | Ensure that the page, or at least one of its frames contains an level one heading | best-practice | true |
```

Comment:
"...contains _a level-one_ heading"

Note both the "an" to "a" change _and_ the added hyphen in "level-one" (it's acting as an adjective here).